src/ctelnet.cpp

```diff
@@ -356,28 +356,24 @@ void cTelnet::handle_socket_signal_disconnected()
 void cTelnet::handle_socket_signal_hostFound(QHostInfo hostInfo)
 {
     if (!hostInfo.addresses().isEmpty()) {
-        mHostAddress = hostInfo.addresses().first();
-        QString msg = "[ INFO ]  - The IP address of " + hostName + " has been found. It is: " + mHostAddress.toString() + "\n";
-        postMessage(msg);
-        msg = "[ INFO ]  - Trying to connect to " + mHostAddress.toString() + ":" + QString::number(hostPort) + " ...\n";
-        postMessage(msg);
+        mHostAddress = hostInfo.addresses().constFirst();
+        postMessage(tr("[ INFO ]  - The IP address of %1 has been found. It is: %2\n").arg(hostName, mHostAddress.toString()));
+        postMessage(tr("[ INFO ]  - Trying to connect to %1: %2 ...\n").arg(mHostAddress.toString(), QString::number(hostPort)));
         socket.connectToHost(mHostAddress, hostPort);
     } else {
         socket.connectToHost(hostInfo.hostName(), hostPort);
-        QString msg = "[ ERROR ] - Host name lookup Failure!\nConnection cannot be established.\nThe server name is not correct, not working properly,\nor your nameservers are not working properly.";
+        QString msg = tr("[ ERROR ] - Host name lookup Failure!\nConnection cannot be established.\nThe server name is not correct, not working properly,\nor your nameservers are not working properly.");
```

Comment:
Actually - there is no need to create the `msg` local variable when the `tr(`...`)` can be placed directly as the argument to the `postMessage(`...`)` call - like the iines in the other branch of the `if (`...`)`. 😁